Gruntfile.js

```diff
@@ -34,7 +34,8 @@ module.exports = function (grunt) {
 
         extract_cldr_data: {
             options: {
-                pluralRules: true
+                pluralRules: true,
+                numberFields: true
```

Comment:
hmmm.  Seems this will increase the bundle size quite a bit...

Reply:
Thanks for calling that out.

For our ember bundles, we'll likely make this off by default and expose as a config option.

Reply:
@jasonmit Yeah that is a good point!  The only thing I noticed is we are duplicating `pluralRuleFunction` for a particular lang.  One in the vendor dist and the other in the app bundle.  

I'm not 100% sure the locale data for this lib is needed in the vendor.js file, but perhaps you know much better than I do.  My understanding was that at runtime, we handoff the relevant data generated from broccoli-cldr-data to `IntlMessageFormat`.  Don't see us `require`ing this data anywhere.  Lmk your thoughts!

- https://github.com/ember-intl/ember-intl/blob/0eda54c7a1784ebca54b4e72e134bd53e9de2457/addon/hydrate.js#L36
- https://github.com/ember-intl/ember-intl/blob/master/addon/services/intl.js#L182
- https://github.com/ember-intl/intl-messageformat/blob/master/src/core.js#L142

Reply:
> The only thing I noticed is we are duplicating pluralRuleFunction for a particular lang. One in the vendor dist and the other in the app bundle.

Sounds like a bug, this is happening on master & easy to reproduce?  If so, will investigate soon.

Reply:
Oh you are right!  Other languages does not add anything to the vendor output.  👍